packages/libs/eda/src/lib/map/index.ts

```diff
@@ -12,4 +12,4 @@ export type SiteInformationProps = {
 export const mapNavigationBackgroundColor = 'white';
 export const mapNavigationBorder: CSSProperties['border'] = '1px solid #D9D9D9';
 export const mapSideNavigationActiveMenuItemBackground: CSSProperties['background'] =
-  '#E1F5FF';
+  '#ECF7F9';
```

Comment:
**Question:** should this value come from the theme palette? Not sure if we have something comparable.

Reply:
Whoops! I forgot to delete that! It was a dangler placeholder!! The component uses the theme hook!

https://github.com/VEuPathDB/web-monorepo/pull/31/commits/3a86b1c56dca486bc86d3fc28de4e41b0ff06834